packages/client/hmi-client/src/components/model/model-parts/tera-model-part-entry.vue

```diff
@@ -19,6 +19,19 @@
 				<!--amr_to_mmt doesn't like unit expressions with spaces, removing them here before they are saved to the amr-->
 				<template v-else-if="showUnit">
 					<template v-if="featureConfig.isPreview"><label>Unit</label>{{ item.unitExpression }}</template>
+					<!-- we use a dropdown for units with time semantic-->
+					<Dropdown
+						v-else-if="isTimePart"
+						:model-value="item.unitExpression"
+						option-label="label"
+						option-value="value"
+						:options="[
```

Comment:
Likely will want these as an enum in future sections of this
For this section this is not required

Reply:
Agreed - I think I might do this in part 4 as some final cleanups